numerics/root_finders_test.cpp

```diff
@@ -53,6 +56,37 @@ TEST_F(RootFindersTest, SquareRoots) {
   }
 }
 
+TEST_F(RootFindersTest, GoldenSectionSearch) {
```

Comment:
Add a test with a function that has multiple extrema on the search interval.

Reply:
Done.